levels/spriteDance.js

```diff
@@ -7,13 +7,14 @@ module.exports = {
       whenSetup(function () {
         lead_dancer = makeNewDanceSprite("CAT", lead_dancer, {x: 200, y: 200});
       });
-      
+
       everySeconds(2, "measures", function () {
         changeMoveLR(lead_dancer, 3, -1);
       });
-      
+
       everySeconds(4, "measures", function () {
         changeMoveLR(lead_dancer, 1, -1);
+        setProp(lead_dancer, "scale", 50);
```

Comment:
Spent a bunch of time trying to fire this event in the event immediately above, before discovering https://github.com/code-dot-org/dance-party/issues/165.